src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
```

Comment:
and last but not least, remember the enum with colors you created? use it here instead of just String to avoid inconsistencies